### DISCUSSION TITLE
"Freeze/Unfreeze tracks" commands?

### DISCUSSION THREAD

**Discussion by ll-dasein-ll** [KBLiveSolutions/open.control#18] [Feature Requests & Ideas]
Hi Pierre -

Could Freeze/Unfreeze Selected Tracks be added as a button command?  I've just thought how useful that would be, while I'm sitting here having to freeze some tracks right now, and I can't see it as an option.

Cheers,
Martin

**Comment by KBLiveSolutions** [comment]
Good idea, but unfortunately this is not accessible through the API.

**Reply by ll-dasein-ll** [reply]
That's a shame...  Ah well, thanks for looking 😊
